force_wfmanager/left_side_pane/new_entity_modal.py

```diff
@@ -91,37 +83,37 @@ class NewEntityModal(HasStrictTraits):
     #: Disable the OK button if no factory set
     OKCancelButtons[0].trait_set(enabled_when="selected_factory is not None")
 
-    model_description_HTML = Property(HTML, depends_on="current_model")
+    model_description_HTML = Property(Unicode, depends_on="current_model")
 
-    current_model_editable = Property(Bool, depends_on='current_model')
+    current_model_editable = Property(Bool, depends_on="current_model")
 
-    no_config_options_msg = ReadOnly(HTML)
+    no_config_options_msg = ReadOnly(Unicode)
```

Comment:
uhm... I am not sure, but I think that if we use HTML, we can skip using HTMLEditor() in the view. Do you have more information from Corran about possible different behavior? I used Unicode, but I might be wrong.

Reply:
I haven't asked Corran, but in trait_types.py it's a subclass of String, with some metadata saying to use HTMLEditor. 
I was thinking it would be useful to have it as unicode, but I'll check whether this is a good idea!